validator/gke_cluster_location_v2_test.rego

```diff
@@ -0,0 +1,249 @@
+#
```

Comment:
We don't need to calls this file `_v2` because only the latest version of constraints is maintained/actively updated.